lib/ruby_ui/codeblock/codeblock.rb

```diff
@@ -54,19 +54,7 @@ def with_clipboard
     def codeblock
       div(**attrs) do
         div(class: "after:content-none") do
-          pre do
-            current_version = Gem.loaded_specs["phlex"].version
-
-            if current_version.segments[0] >= 2
-              # Any 2.x version (including betas)
-              raw(safe(FORMATTER.format(lexer.lex(@code))))
-            else
-              # Any 1.x version
-              unsafe_raw FORMATTER.format(
-                lexer.lex(@code)
-              )
-            end
-          end
+          pre { raw(safe(FORMATTER.format(lexer.lex(@code)))) }
```

Comment:
I'm not sure if is recommended to flag code as safe by default. Our users can pass user generated content here and create a security breach. I think we should force the users to use `safe` when passing content, so they will know what they are doing.

Reply:
Yeah agreed but in this context is it not safe because we are limiting what is being passed as safe? It's only in the code component to render it.

Reply:
Good question. I will make some tests here.

Reply:
It’s an interesting debate. It may be cumbersome to require the user to call safe by himself on a code block. A code block component implies that the code will be interpreted. Let's keep that way for now.

Reply:
@SethHorsley @cirdes I think Rouge is escaping the code, so I think it is safe since we are using safe on rouge result and not on given code.